.github/workflows/update-readme.yml

```diff
@@ -20,7 +20,7 @@ jobs:
        tmpfile=$(mktemp)
        unset latest
        unset stable
-       for rancher_major_minor in 2.7 2.6 2.5; do
+       for rancher_major_minor in 2.7 2.6; do
```

Comment:
Looks like this was done already: https://github.com/rancher/rancher/blob/release/v2.8/.github/workflows/update-readme.yml#L23